plugins/chat/assets/javascripts/discourse/components/chat-thread.gjs

```diff
@@ -528,6 +543,106 @@ export default class ChatThread extends Component {
     return prev;
   }
 
+  get membership() {
+    return this.args.thread.currentUserMembership;
+  }
+
+  @action
+  createThreadMembership() {
+    if (this.membership || !this.currentUser.admin) {
+      return;
+    }
+
+    this.args.thread.currentUserMembership = UserChatThreadMembership.create({
```

Comment:
I dont understand why this is necessary?

Reply:
When a thread is created, the original message user automatically has a thread membership, however the admin users won't have membership. We want to update the thread_title_prompt_seen value but their is no record for admins.

Reply:
I will just apply censoring again from the serializer for now anyway

Reply:
Maybe we just don't show it for admins? Admins are supposed to know they can do this kind of things...

Reply:
If we are okay to change the requirements a little then we could safely remove this (currently it is "if the OP or an admin views an untitled thread"). I agree that it would be nicer if we didn't have to create the membership here.

Reply:
Ok I see, let me think about it.